godtools/App/Features/Dashboard/Presentation/Tools/ToolsViewModel.swift

```diff
@@ -67,34 +67,45 @@ class ToolsViewModel: ObservableObject {
             .getLanguagePublisher()
             .assign(to: &$appLanguage)
         
-        Publishers.CombineLatest(
-            $appLanguage.eraseToAnyPublisher(),
-            $toolFilterLanguageSelection.eraseToAnyPublisher()
-        )
-        .flatMap({ (appLanguage: AppLanguageDomainModel, toolFilterLanguage) -> AnyPublisher<(ViewToolsDomainModel, [SpotlightToolListItemDomainModel]), Never> in
-            
-            let selectedLanguageFilter = toolFilterLanguage.language
-            
-            return Publishers.CombineLatest(
-                viewToolsUseCase.viewPublisher(translatedInAppLanguage: appLanguage, languageForAvailabilityText: selectedLanguageFilter),
-                getSpotlightToolsUseCase.getSpotlightToolsPublisher(translatedInAppLanguage: appLanguage, languageForAvailabilityText: selectedLanguageFilter)
-            )
-            .eraseToAnyPublisher()
-        })
-        .receive(on: DispatchQueue.main)
-        .sink { [weak self] (domainModel: ViewToolsDomainModel, spotlightTools: [SpotlightToolListItemDomainModel]) in
+        $appLanguage.eraseToAnyPublisher()
+            .flatMap { (appLanguage: AppLanguageDomainModel) -> AnyPublisher<(AppLanguageDomainModel, UserToolFiltersDomainModel), Never> in
                 
-            self?.favoritingToolBannerMessage = domainModel.interfaceStrings.favoritingToolBannerMessage
-            self?.toolSpotlightTitle = domainModel.interfaceStrings.toolSpotlightTitle
-            self?.toolSpotlightSubtitle = domainModel.interfaceStrings.toolSpotlightSubtitle
-            self?.filterTitle = domainModel.interfaceStrings.filterTitle
-            
-            self?.spotlightTools = spotlightTools
-            
-            self?.allTools = domainModel.tools
-            self?.isLoadingAllTools = false
-        }
-        .store(in: &cancellables)
+                return getUserToolFiltersUseCase.getUserToolFiltersPublisher(translatedInAppLanguage: appLanguage)
```

Comment:
Just wondering, did you change this to call the `getUserToolFiltersPublisher` directly rather than listening for updates from `$toolFilterLanguageSelection.eraseToAnyPublisher()` because it'll likely use the dummy initial value the first time this chain evaluates?

Reply:
Hey @rachaelblue no, I would have left the ```$toolFilterLanguageSelection.eraseToAnyPublisher()```, but realized I also needed the category filter to push into the UseCase which is what caused this change. 

I'm glad you mentioned this because now that I think about it, it would be better to create a private variable for the toolFilterCategorySelection and include that rather than calling the use case again. Going to update this.

Reply:
That's another good point about the dummy initial value. I would like to keep that in the business logic, hmm.  The app language is also set to english by default.  Maybe I should just make those empty string values.

Reply:
1 more change, I went ahead and set the category and language filter selection text to empty strings just for the sake of keeping as much as we can in the business logic.